src/strongbox/core.clj

```diff
@@ -179,108 +170,6 @@
   [& path]
   (nav-map (get themes (get-state :cfg :gui-theme)) path))
 
-;; db
-
-(defn as-unqualified-hyphenated-maps
-  "used to coerce keys in each row of resultset
-  adapted from https://github.com/seancorfield/next-jdbc/blob/master/doc/result-set-builders.md#rowbuilder-and-resultsetbuilder"
-  [rs opts]
-  (let [xform #(-> % name lower-case (clojure.string/replace #"_" "-"))
-        unqualified (constantly nil)]
-    (rs/as-modified-maps rs (assoc opts :qualifier-fn unqualified :label-fn xform))))
-
-(defn get-db
```

Comment:
add the db ns to test

Reply:
+ bump coverage past threshold

Reply:
profiling is causing the profiled sections to be ignored by coverage.

this is because it's outside of 'debug mode'. so ... hm.

Reply:
possibly a custom test runner that does some init and then calls the standard clojure.test runner: 
https://github.com/cloverage/cloverage/blob/master/cloverage/src/cloverage/coverage.clj#L153-L164

Reply:
done! it was awkward but profiled forms are now being counted properly and usage of spec checking and profiling is now much clearer